Readme.md

```diff
@@ -3,61 +3,67 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3252)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to bind the GridView with the Entity Framework in a regular and Database Server modes
 
+# Grid View for ASP.NET Web Forms - How to bind grid to Entity Framework in regular and database server modes
 
-<p>This example is an illustration of the <a href="https://www.devexpress.com/Support/Center/p/KA18615">KA18615: How to bind MVC GridView Extension with Entity Framework in a server mode</a> KB Article. Please refer to the article for an explanation.<br />
-This example illustrates how to use the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebMvcGridViewExtension_BindToEFtopic"><u>GridViewExtension.BindToEF</u></a> method for binding the MVC GridView Extension with the EF data content in:</p><p>- The <a href="http://documentation.devexpress.com/#AspNet/CustomDocument14580"><u>Grid mode</u></a>. All data shaping operations are performed on the WebServer/Grid side (for a small volume of data). Note that a table from the EF context used as the grid's Model should be explicitly evaluated, for example, via the <strong>ToList</strong> method:</p>
+This example demonstrates two approaches how to bind a grid to Entity Framework. The first approach is recommended for small data sources because it downloads the data to the grid. The second approach operates on the database side and is recommended for large amount of data.
```

Comment:
```suggestion
This example demonstrates two approaches on how to bind a grid to Entity Framework. The first approach is recommended for small data sources because it downloads the data to the grid. The second approach operates on the database side and is recommended for large amounts of data.
```